libtransmission/announcer.cc

```diff
@@ -669,11 +659,13 @@ static tr_tier* getTier(tr_announcer_impl* announcer, tr_sha1_digest_t const& in
     return tor->torrent_announcer->getTier(tier_id);
 }
 
-/***
-****  PUBLISH
-***/
+/// PUBLISH
```

Comment:
Should be:
```
#pragma mark PUBLISH
```

Reply:
This is the same discussion as above, so let's figure it out there & I'll mark this thread as resolved